### DISCUSSION TITLE
How do I change the Title Screen layout (FNF Bumping Logo and GF)?

### DISCUSSION THREAD

**Discussion by AKRGames** [CodenameCrew/CodenameEngine#118] [Q&A]
I've been looking at the TitleState.hx file, and neither of the functions `startIntro()` or `skipIntro()` says nothing about the title screen layout after the intro texts play out. I also observed that the variables `logoBl`, `gfDance`, and `danceLeft` hasn't been used.
With that said, if I were to remove GF from the title screen and only keep the logo, how would I do it?

**Comment by mario2018221** [comment]
Hello,
The title screen uses an XML file.
If you want to remove GF and only keep the logo, extract this .zip file. And go to your Codename Build (or source code) and go to:
'assets/data/titlescreen', and replace the xml with the one that's in the zip file, and that should work.

[The ZIP file in question](https://github.com/FNF-CNE-Devs/CodenameEngine/files/11413720/titlescreenWithoutGF.zip)

Hopefully this helped.
